packages/static/src/progress-bar/styles/progress-bar.module.css

```diff
@@ -0,0 +1,109 @@
+@import '../../theme/theme.module.css';
+
+@keyframes progressLoading {
+  from {
+    background-position-x: 200%;
+  }
+  to {
+    background-position-x: 0;
+  }
+}
+
+@keyframes progressSlideLoading {
+  0% {
+    left: 0%;
+    right: 100%;
+    width: 0%;
+  }
+  10% {
+    left: 0%;
+    right: 75%;
+    width: 25%;
+  }
+  90% {
+    right: 0%;
+    left: 75%;
+    width: 25%;
+  }
+  100% {
+    left: 100%;
+    right: 0%;
+    width: 0%;
+  }
+}
+
+.progressBarWrapper {
+  width: 100%;
+}
+
+.progressBarLabel {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: var(--spacing-2);
+}
+
+.progressBarLabelSpan {
+  font-weight: var(--font-weight-normal);
+  font-size: var(--font-size-label1);
+}
+
+/* Variants for "variant" */
+.variantRounded {
+  border-radius: var(--cdg-border-radius-full);
+}
+
+.variantRounded div {
+  border-radius: var(--cdg-border-radius-full);
+}
+
+/* Variants for "size" */
+.sizeSm {
+  height: var(--cdg-spacing-0_5);
+}
+
+.sizeMd {
+  height: var(--cdg-spacing-1);
+}
+
+.sizeLg {
+  height: var(--cdg-spacing-2);
+}
+
+.sizeXl {
+  height: var(--cdg-spacing-3);
+}
+
+.progressBar {
+  position: relative;
+}
+
+.currentProgress {
+  height: 100%;
+  transition: width 0.2s linear;
+}
+
+.loadingSlide {
+  /* styles for loading: slide would go here */
+}
+
+.loadingProgress {
+  position: absolute;
+  inset: 0;
+  height: 100%;
+}
+
+/* Variants for "loading" */
+.loadingStripes {
+  width: 100%;
+  background-size: 200% 100%;
+  animation: progressLoading 10s linear infinite;
+  will-change: background-position;
+}
+
+.loadingTrue {
+  animation: progressSlideLoading 2s linear infinite;
+}
+
+.loadingFalse {
+  display: none;
```

Comment:
should we unmount the element instead of display none?

Reply:
fixed